File: Java/TestingSystem_Assignment_10/TestingSystem_Assignment_10/src/main/java/org/example/vti/entity/Account.java
package org.example.vti.entity;

import java.sql.Date;

public class Account {
    private int id;
    private String email;
    private String name;
    private String fullName;
    private int department;
    private int position;
    private Date createDate;

    public Account(){
    }


        public Account(int id, String email, String name, String fullName, int department, int position, Date createDate) {
        this.id = id;
        this.email = email;
        this.name = name;
        this.fullName = fullName;
        this.department = department;
        this.position = position;
        this.createDate = createDate;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getFullName() {
        return fullName;
    }

    public void setFullName(String fullName) {
        this.fullName = fullName;
    }

    public int getDepartment() {
        return department;
    }

    public void setDepartment(int department) {
        this.department = department;
    }

    public int getPosition() {
        return position;
    }

    public void setPosition(int position) {
        this.position = position;
    }

    public Date getCreateDate() {
        return createDate;
    }

    public void setCreateDate(Date createDate) {
        this.createDate = createDate;
    }

    @Override
    public String toString() {
        return "Account{" +
                "id=" + id +
                ", email='" + email + '\'' +
                ", name='" + name + '\'' +
                ", fullName='" + fullName + '\'' +
                ", department=" + department +
                ", position=" + position +
                ", createDate=" + createDate +
                '}';
    }
}

File: ReactJS/TestingSystem_Assignment_1/ex1/ex1DataType/Account.ts
import { Department } from "./Department";
import { Position } from "./Positon";
export class Account{

//   AccountID: định danh của User (auto increment)
//  Email:
//  Username:
//  FullName:
//  DepartmentID: phòng ban của user trong hệ thống
//  PositionID: chức vụ của User
//  CreateDate: ngày tạo tài khoản

    private id: number;
    private email: string;
    private username: string;
    private fullName: string;
    private department: Department;
    private position: Position;
    private createDate: Date;

    constructor() {
	}


    /**
     * Getter id
     * @return {number}
     */
	public getId(): number {
		return this.id;
	}

    /**
     * Getter email
     * @return {string}
     */
	public getEmail(): string {
		return this.email;
	}

    /**
     * Getter username
     * @return {string}
     */
	public getUsername(): string {
		return this.username;
	}

    /**
     * Getter fullName
     * @return {string}
     */
	public getFullName(): string {
		return this.fullName;
	}

    /**
     * Getter department
     * @return {Department}
     */
	public getDepartment(): Department {
		return this.department;
	}

    /**
     * Getter position
     * @return {Position}
     */
	public getPosition(): Position {
		return this.position;
	}

    /**
     * Getter createDate
     * @return {Date}
     */
	public getCreateDate(): Date {
		return this.createDate;
	}

    /**
     * Setter id
     * @param {number} value
     */
	public setId(value: number) {
		this.id = value;
	}

    /**
     * Setter email
     * @param {string} value
     */
	public setEmail(value: string) {
		this.email = value;
	}

    /**
     * Setter username
     * @param {string} value
     */
	public setUsername(value: string) {
		this.username = value;
	}

    /**
     * Setter fullName
     * @param {string} value
     */
	public setFullName(value: string) {
		this.fullName = value;
	}

    /**
     * Setter department
     * @param {Department} value
     */
	public setDepartment(value: Department) {
		this.department = value;
	}

    /**
     * Setter position
     * @param {Position} value
     */
	public setPosition(value: Position) {
		this.position = value;
	}

    /**
     * Setter createDate
     * @param {Date} value
     */
	public setCreateDate(value: Date) {
		this.createDate = value;
	}

}